boards/BMS/Src/App/states/App_ChargeState.c

```diff
@@ -25,16 +27,18 @@ static void ChargeStateRunOnTick100Hz(struct StateMachine *const state_machine)
 {
     if (App_AllStatesRunOnTick100Hz(state_machine))
     {
-        struct BmsWorld *         world       = App_SharedStateMachine_GetWorld(state_machine);
-        struct BmsCanTxInterface *can_tx      = App_BmsWorld_GetCanTx(world);
-        struct Charger *          charger     = App_BmsWorld_GetCharger(world);
-        struct Accumulator *      accumulator = App_BmsWorld_GetAccumulator(world);
-        struct Airs *             airs        = App_BmsWorld_GetAirs(world);
+        struct BmsWorld *            world   = App_SharedStateMachine_GetWorld(state_machine);
+        struct BmsCanTxInterface *   can_tx  = App_BmsWorld_GetCanTx(world);
+        struct BmsCanRxInterface *   can_rx  = App_BmsWorld_GetCanRx(world);
+        struct Charger *             charger = App_BmsWorld_GetCharger(world);
+        struct Airs *                airs    = App_BmsWorld_GetAirs(world);
+        struct TractiveSystem *const ts      = App_BmsWorld_GetTractiveSystem(world);
 
         static uint16_t ignore_chgr_fault_counter      = 0U;
```

Comment:
For viz: This counter doesn't look like it actually does anything since its redefined to 0 at every time this function is called. Out of scope of this PR but can be addressed in another ticket